base/pkg/entry.jl

```diff
@@ -174,13 +174,13 @@ function status(io::IO, pkg::AbstractString, ver::VersionNumber, fix::Bool)
             isfile("METADATA",pkg,"url") || push!(attrs,"unregistered")
             LibGit2.isdirty(prepo) && push!(attrs,"dirty")
             isempty(attrs) || print(io, " (",join(attrs,", "),")")
-        catch
-            print(io, "broken-repo (unregistered)")
+        catch err
```

Comment:
makes sense, I prefer this over https://github.com/JuliaLang/julia/issues/13639 so +1